config/webpack/environment.js

```diff
@@ -1,11 +1,27 @@
 const { environment } = require('@rails/webpacker');
 
+// This file is similar to what what you'd have
+// inside webpack.config.js for a Node project.
+//
+// See also babel.config.js
+//
+// See docs at https://webpack.js.org/configuration/
+
 // Ignore JS test files.
 environment.loaders.append('ignore', {
   test: /\.test\.js$/,
   loader: 'ignore-loader',
 });
 
+// Transpile Enketo files.
```

Comment:
This supports a big breaking change in enketo so now we can compile their modern syntax. Previously, they shipped code that was fully transpiled already and we didn't need to take this step.

Reply:
hmm wonder why they did that?